sandbox/lua/modules/graphite.lua

```diff
@@ -0,0 +1,135 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
+-- License, v. 2.0. If a copy of the MPL was not distributed with this
+-- file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+--[[
+Module contains utilities for Graphite
+--]]
+-- Imports
+require "string"
+require "math"
+require "table"
+
+local M = {}
+
```

Comment:
Okay, then i'll update  PR (with docstrings)  later

On Thu, Apr 23, 2015 at 4:10 PM, Mike Trinkala <notifications@github.com>
wrote:

> In sandbox/lua/modules/graphite.lua
> <https://github.com/mozilla-services/heka/pull/1492#discussion_r28965410>:
>
> > @@ -0,0 +1,135 @@
> > +-- This Source Code Form is subject to the terms of the Mozilla Public
> > +-- License, v. 2.0. If a copy of the MPL was not distributed with this
> > +-- file, You can obtain one at http://mozilla.org/MPL/2.0/.
> > +
> > +--[[
> > +Module contains utilities for Graphite
> > +--]]
> > +-- Imports
> > +require "string"
> > +require "math"
> > +require "table"
> > +
> > +local M = {}
> > +
>
> I was playing with some code to address 1496 but the original hack of not
> using environments will have to stand for now.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla-services/heka/pull/1492/files#r28965410>.
>



-- 
jarek@reijutsu:~$ fortune
You have Egyptian flu: you're going to be a mummy.
jarek@reijutsu:~$ fortune
You now have Asian Flu.